azure-pipelines.yml

```diff
@@ -0,0 +1,12 @@
+#

```

Comment:
I think the leading and trailing `#` should be dropped, and instead separated with a newline - unless there is some auto-formatter fighting against this.